snuba/stateful_consumer/states/consuming.py

```diff
@@ -14,9 +14,13 @@ class ConsumingState(State[ConsumerStateCompletionEvent, ConsumerStateData]):
     starts.
     """
 
-    def __init__(self, consumer: BatchingKafkaConsumer) -> None:
+    def __init__(
+        self,
+        consumer_builder: ConsumerBuilder,
+    ) -> None:
         super(ConsumingState, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```

(I think this is addressed in one of the prior PRs, but just in case.)